cmd/tailscaled/tailscaled_windows.go

```diff
@@ -299,6 +300,14 @@ func beWindowsSubprocess() bool {
 		}
 	}()
 
+	// Pre-load wintun.dll using a fully-qualified path so that wintun-go
+	// load our copy and not some (possibly outdated) copy dropped in system32.
+	// (OSS Issue #10023)
+	fqWintunPath := fullyQualifiedWintunPath(log.Printf)
+	if _, err := windows.LoadDLL(fqWintunPath); err != nil {
+		log.Printf("Error pre-loading \"%s\": %v", fqWintunPath, err)
```

Comment:
Nitpick, feel free to ignore:
```suggestion
		log.Printf("Error pre-loading %q: %v", fqWintunPath, err)
```

Reply:
I intentionally didn't use `q` because I don't like the escaped backslashes in logs. Personal preference. :-)

Reply:
👍

Reply:
%#q is another option if you want: https://go.dev/play/p/2_oEKM1fKp8

Reply:
Huh, TIL; that's good to know 👍